### ISSUE TITLE
Minion level on Raise Spectre gem incorrectly applies to Phantasms from Summon Phantasm Support

### ISSUE THREAD

**Issue by Quotae** [Openarl/PathOfBuilding#1839]
When using Summon Phantasm Support with Raise Spectre, the level of the Spectre minions on the gem is applying to the phantasms. The wording on Raised Spectre is "Spectres are level ___", which should mean that phantasms ignore that line and instead be based on the level requirement of the gem.

Here is a build (not mine) that shows this behavior. Raise Spectre and Ethereal Knives both require level 70, but the Phantasms from Raise Spectre are showing significantly more base physical damage. https://pastebin.com/3UYAqze2

If you set Raise Spectre to level 16, then the summoned spectres will be level 70. This makes the phantasms from EK and Raise Spectre have the same damage.

I think the reason for this is that the Raise Spectre gem has this mod on it:
![image](https://user-images.githubusercontent.com/39030429/72305408-48c30e80-3639-11ea-8604-e0701690935a.png)

Since this is applying to the entire skill, it's also applying to the phantasm support.

This is a pretty niche case and isn't even one that I personally noticed, but someone on Discord asked about it and this is what I found after investigating it.

**Comment by ZzZombo** [comment]
Isn't it intended? I recall the wiki stating that the stats of the summoned phantasms are based on then supported gem.

**Comment by deathbeam** [comment]
Yeah, based on level of active skill, but this issue is about phantasms taking level of raise spectre minion and not raise spectre gem.
